### ISSUE TITLE
Press release collection formatting

### ISSUE THREAD

**Issue by lpgoldstein** [DOI-ONRR/onrr.gov-site#2773]
Press release collection items should be formatted so that the titles are headers. The items in the collection read as text. 
https://onrr.gov/about/public-affairs

**Comment by Maroyafaied** [comment]
Wait until text editor is enhanced
